lib/db.js

```diff
@@ -573,6 +592,48 @@ module.exports = (
       }))
   }
 
+  DB.prototype.pruneSessionTokens = function (uid, sessionTokens) {
+    log.trace({ op: 'DB.pruneSessionTokens', uid, tokenCount: sessionTokens.length })
+
+    if (! redis || ! TOKEN_PRUNING_ENABLED || ! features.isLastAccessTimeEnabledForUser(uid)) {
```

Comment:
side note, but are we at the point where we can remove this feature flag and assume it's enabled for all users all the time?

Reply:
Possibly, although I do recall that @jrgm used it when we wanted to safely disable Redis so he could scale up the instance size. If there's a chance we'll want to do that again, it might be worth keeping?